etc/code-samples-action.py

```diff
@@ -0,0 +1,14 @@
+import json
+
+f = open("./examples/apis.json")
+data = json.load(f)
+f.close()
+f = open("code-samples-warning.md", "w")
+f.write(
+    "Warning changing any of the following functions will break code samples on app if an api for these function changes please contact the fleet team\n"
+)
+f.write("|component|function|\n")
+f.write("|-|-|\n")
+for k, v in data.items():
+    f.write(f"|{k}|{v}|\n")
+f.close()
```

Comment:
```suggestion
with open("code-samples-warning.md", "w") as f:
    f.write(
        "Warning changing any of the following functions will break code samples on app if an api for these function changes please contact the fleet team\n"
    )
    f.write("|component|function|\n")
    f.write("|-|-|\n")
    for k, v in data.items():
        f.write(f"|{k}|{v}|\n")
```

Reply:
used the fancy file opens in my latest commit